packages/lib-classifier/src/plugins/drawingTools/experimental/components/LineControls/LineControls.js

```diff
@@ -35,7 +35,8 @@ const LineControls = forwardRef(function LineControls({
   const { canvas, rotate } = useContext(SVGContext)
 
   // Get the transformation that's been applied to a parent element
-  const { matrix } = canvas.parentElement.parentElement.transform.baseVal.consolidate()
+  const canvasSVG = canvas.closest('svg')
+  const matrix = canvasSVG.firstChild.getCTM()
```

Comment:
`VisXZoom` applies a scaling and translation transform to the first child of the `SingleImageCanvas` SVG. This is less brittle than `canvas.parentElement.parentElement`, but still couples `LineControls` to the internal implementation of `SingleImageCanvas`, which is not ideal.